components/usage/pkg/controller/billing.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package controller
+
+import "github.com/gitpod-io/gitpod/usage/pkg/stripe"
+
+type BillingController interface {
+	SubmitUsageToStripe(report []TeamUsage)
```

Comment:
For it to remain a generic interface that makes sense, it shouldn't mention a concrete implementation. Here, it should probs be 
```suggestion
	SubmitReport(report []TeamUsage)
```

Or even just `Reconcile()`

Reply:
[5eafb9b](https://github.com/gitpod-io/gitpod/pull/10754/commits/5eafb9bab1435d646c77a2ee57393d474f96c3d0)